sdcm/utils/aws_builder.py

```diff
@@ -114,104 +114,59 @@ def name(self):
     def jenkins_labels(self):
         return f"aws-sct-builders-{self.region.region_name}-v2-asg"
 
-    @cached_property
-    def instance(self) -> EC2ServiceResource.Instance:
-
-        if not self.runner.image:
-            LOGGER.error("SCT Runner image was not found in %s! "
-                         "Use `hydra create-runner-image --cloud-provider %s --region %s'",
-                         self.region.region_name, self.runner.CLOUD_PROVIDER, self.region.region_name)
-            return None
-
-        instances = self.region.client.describe_instances(Filters=[{"Name": "tag:Name",
-                                                                    "Values": [self.name]},
-                                                                   {"Name": "instance-state-name",
-                                                                    "Values": ["running"]}])
-
-        if instances["Reservations"] and instances["Reservations"][0]["Instances"]:
-            return self.region.resource.Instance(instances["Reservations"][0]["Instances"][0]['InstanceId'])
-
-        tags = {
-            "bastion": "true",
```

Comment:
so we no longer use builders as bastions?

Reply:
we use the sct-runner available on that region, if we'll see it's not enough, we'll thing if we want few machines like that, but they won't be connect to jenkins.

Reply:
in GCP so far we don't have any, no one complain about it. (so far)